__test__/unit/adapters/simple_interest_loan_adapter.spec.ts

```diff
@@ -497,6 +505,61 @@ describe("Simple Interest Loan Adapter (Unit Tests)", async () => {
         });
     });
 
+    describe("#getRepaymentSchedule", () => {
+        let simpleInterestTermsContractAddress: string;
+        let principalToken;
+        let termsContract;
+
+        beforeAll(async () => {
+            const termsContractRegistry = await TermsContractRegistryContract.deployed(
+                web3,
+                TX_DEFAULTS,
+            );
+            simpleInterestTermsContractAddress = await termsContractRegistry.getSimpleInterestTermsContractAddress.callAsync(
+                principalTokenAddress,
+            );
+
+            principalToken = principalTokenAddress;
+            termsContract = simpleInterestTermsContractAddress;
+        });
+
+        describe("when the terms length is 2", () => {
+            describe("when the amortization unit is 'weeks'", () => {
+                test("it returns a list of 2 unix timestamps 1 week apart", async () => {
+                    const issuanceTime = moment().unix();
+
+                    const contractTermsParameters =
+                        "0x00000000000000000fd217f5c3f2000002000000000000000000000000000002";
```

Comment:
It feels like we should be using the `packParameters` function here to generate this string, as opposed to just hardcoding a magic string.  Would make things less brittle / more readable IMO.